deep_speech_2/README.md

```diff
@@ -283,28 +283,36 @@ The hyper-parameters $\alpha$ (coefficient for language model scorer) and $\beta
     CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 \
     python tools/tune.py \
     --trainer_count 8 \
-    --alpha_from 0.1 \
-    --alpha_to 0.36 \
-    --num_alphas 14 \
-    --beta_from 0.05 \
-    --beta_to 1.0 \
-    --num_betas 20
+    --alpha_from 1.0 \
+    --alpha_to 3.2 \
+    --num_alphas 45 \
+    --beta_from 0.1 \
+    --beta_to 0.45 \
+    --num_betas 8
     ```
 
 - Tuning with CPU:
 
     ```bash
     python tools/tune.py --use_gpu False
     ```
+ The grid search will log the WER (word error rate) or CER (character error rate) at each point in the hyper-parameter space and their minima, and draw the error surface optionally. A proper hyper-parameters range should include the global minima of the error surface for WER/CER, as illustrated in the following figure.  
```

Comment:
log-->print ?
remove `and their minima`.

Reply:
Done